src/app/view/perspectives/post-it/document/post-it-document.component.html

```diff
@@ -1,135 +1,65 @@
+          [model]="row.newValue || row.value"
+          (remove)="onUpdateRow(idx, row.newName || row.name, '')"
+          (update)="onUpdateRow(idx, row.newName || row.name, $event)"
+          (focus)="selectionHelper.focusCellSpan(idx, 1, getDocumentKey())"
+          (enter)="selectionHelper.moveToNextInput()">
+        </post-it-document-cell>
       </li>
 
-      <li>
-        <span id="{{ perspectiveId }}{{ postItModel.index }}[0, {{ postItRows.length }}]"
-              class="attribute"
-              [attr.tabindex]="postItModel.index * 1000 + postItRows.length * 2"
-              (keydown.ArrowLeft)="selectionHelper.moveSelection(-1, 0)"
-              (keydown.ArrowRight)="selectionHelper.moveSelection(1, 0)"
-              (keydown.ArrowUp)="selectionHelper.moveSelection(0, -1)"
-              (keydown.ArrowDown)="selectionHelper.moveSelection(0, 1)"
-              (keydown.F2)="selectionHelper.setEditMode(true); selectionHelper.focus()"
-              (keydown.enter)="selectionHelper.setEditMode(true); selectionHelper.focus()"
-              (focus)="selectionHelper.setEditMode(true); this.selectionHelper.select(0, postItRows.length, postItModel)">
-
-          <input class="border-0" remove-placeholder-on-focus #newAttributeInput
-                 placeholder=" + "
-                 [attr.list]="suggestionListId()"
-                 [(ngModel)]="postItNewRow.attributeName"
-                 (keydown)="$event.stopPropagation()"
-                 (keydown.F2)="selectionHelper.setEditMode(false); selectionHelper.focus()"
-                 (keydown.Escape)="selectionHelper.setEditMode(false); selectionHelper.focus()"
-                 (keydown.enter)="postItNewRow.attributeName? newAttributeInput.blur() : onEnterKeyPressedInEditMode()"
-                 (blur)="postItNewRow.attributeName && createAttributePair()">
-        </span>
-
-        <span>
-        <input class="border-0" [disabled]="true"/>
-        </span>
-      </li>
     </ng-container>
 
-    <ng-container *ngIf="!hasWriteRole()">
-      <li *ngFor="let row of postItRows">
-        <span class="attribute">
-          <input class="border-0"
-                 readonly
-                 [ngModel]="row.attributeName">
-        </span>
 
-        <span>
-          <input class="border-0"
-                 [title]="row.value"
-                 readonly
-                 [ngModel]="row.value">
-        </span>
-      </li>
-    </ng-container>
+    <li class="w-100" *ngIf="hasWriteRole">
+      <div class="w-100 h-100 clickable" (click)="addAttrRow()">
+        <i class="fas fa-plus pt-2 pl-1"></i>
+      </div>
+    </li>
   </ul>
 </div>
 
-<datalist *ngIf="postItModel"
-          id="{{ suggestionListId() }}">
-  <option *ngFor="let attribute of unusedAttributes()">{{ attribute.name }}</option>
+<datalist *ngIf="documentModel" id="{{ suggestionListId() }}">
+  <option *ngFor="let attribute of getUnusedAttributes$() | async">{{ attribute.name }}</option>
```

Comment:
Can we avoid using these functions in templates? It does not help that it returns an observable.